modal/functions.py

```diff
@@ -435,7 +435,7 @@ def _bind_method(
         return fun
 
     @staticmethod
-    def from_args(
+    def from_local(
```

Comment:
Semi-unrelated, but I've been wanting to do this rename, and since I replaced `_from_args` usage in Image I changed this while I was at it...